src/router-setup.js

```diff
@@ -2,29 +2,23 @@ const bodyParser = require('body-parser');
 const config = require('config');
 const Constants = require('./constants');
 const express = require('express');
-const fs = require('fs');
-const Handlebars = require('handlebars');
+const expressHandlebars = require('express-handlebars');
 const mDNSserver = require('./mdns-server');
 const os = require('os');
-const path = require('path');
 const platform = require('./platform');
 const sleep = require('./sleep');
 
-// Build templates
-// eslint-disable-next-line max-len
-Handlebars.registerHelper('escapeQuotes', (str) => new Handlebars.SafeString(str.replace(/'/, '\\\'')));
-
-function getTemplate(name) {
-  const filename = path.join(Constants.VIEWS_PATH, name);
-  return Handlebars.compile(fs.readFileSync(filename, 'utf8'));
-}
-
-const routerSetupTemplate = getTemplate('router-setup.handlebars');
-const creatingTemplate = getTemplate('creating.handlebars');
-const hotspotTemplate = getTemplate('hotspot.handlebars');
+const hbs = expressHandlebars.create({
+  helpers: {
+    escapeQuotes: (str) => `${str}`.replace(/'/, '\\\''),
```

Comment:
This looks like it loses whatever additional escaping HandleBars.SafeString was doing (which might be handled elsewhere)

Reply:
It actually just converts the argument to a string.

https://github.com/wycats/handlebars.js/blob/master/lib/handlebars/safe-string.js

Reply:
Wonderful, a truly Safe string incapable of doing any harm